src/raw/mod.rs

```diff
@@ -2211,11 +2304,70 @@ impl<A: Allocator + Clone> RawTableInner<A> {
         }
     }
 
+    /// Returns an iterator over full buckets indices in the table.
+    ///
+    /// # Safety
+    ///
+    /// The caller to ensure that the `RawTableInner` outlives the
+    /// `FullBucketsIndices`. Because we cannot make the `next` method
+    /// unsafe on the `FullBucketsIndices` struct, we have to make the
+    /// `full_control_bytes_iter` method unsafe.
```

Comment:
By `full_control_bytes_iter` did you mean this method (`full_buckets_indices`)?

Reply:
Done